backend/app/assets/stylesheets/spree/backend/shared/_layout.scss

```diff
@@ -43,14 +43,36 @@
   }
 }
 
+// Page Layout
+//---------------------------------------------------
+.admin-nav {
+  @include position(absolute, 0 null 0 0);
+  width: $width-sidebar;
+}
+
+.main-content {
+  margin-left: $width-sidebar;
+}
+
 // Header
 //---------------------------------------------------
-#header {
+.admin-nav-header {
+  padding: 15px;
+  height: 31px * (0.67); // height of .page-title
+  box-sizing: content-box;
   background-color: $color-1;
-  padding: 5px 0;
-}
+  border-bottom: 1px solid $color-border;
+
+  a {
+    display: inline-block;
+    line-height: 64px;
+  }
 
-#logo { height: 40px }
+  img {
+    vertical-align: middle;
+    margin-top: -12px; // hand-tweaking because the solid logo is bottom heavy
```

Comment:
You should tweak the logo instead. Add some padding in the image, for instance. Lots of shops change the admin logo, so all would have to deal with this. Not ideal IMO.